client/src/components/restaurantInfo.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Styles from './restaurantInfo.module.css';
+// import Logo from './logo.jpg';
+// const logo = require('./logo.jpg'); 
+
+class RestaurantInfo extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state={
+
+        }
+    }
+    render() {
+
+        return (
+            <div className={Styles.restInfo}>
+                <div className={Styles.restName}>
+                    <img  alt="dish" width="100" height="50"></img>
+                RestaurantInfo component<br/>
+                gjh <br/>
+                hgnkjh
+                </div>
+            </div>
+
+        );
+    }
+}
+
+export default RestaurantInfo;
```

Comment:
if you're not using state or lifecycle hooks then refactor to functional component.  Class components are slower and should only be used when necessary.

Reply:
converted to functional component